### ISSUE TITLE
Change configuration path to follow freedesktop specification.

### ISSUE THREAD

**Issue by ainmosni** [owncloud/ocis#5446]
Currently, OCIS writes/reads its configuration from `~/.ocis/config`.

It would be more compliant with modern standards, specifically the [freedesktop specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) for this to live in `${XDG_CONFIG_HOME}/ocis` or, if that variable is not defined, `~/.config/ocis`.

**Comment by butonic** [comment]
I agree, and we should transparently migrate an existing config to the new location, unless we are running in a docker container.

**Comment by IljaN** [comment]
https://github.com/kirsle/configdir
